var/spack/repos/builtin/packages/flang/package.py

```diff
@@ -72,17 +79,30 @@ def post_install(self):
         flang = os.path.join(self.spec.prefix.bin, 'flang')
         with open(flang, 'w') as out:
             out.write('#!/bin/bash\n')
-            out.write(
-                '{0} -I{1} -L{2} {3}{4} -B{5} "$@"\n'.format(
-                    os.path.join(self.spec['llvm'].prefix.bin, 'flang'),
-                    self.prefix.include, self.prefix.lib,
-                    self.compiler.fc_rpath_arg, self.prefix.lib,
-                    self.spec.prefix.bin))
+            if '@develop' in self.spec:
+                out.write(
+                    '{0} -I{1} -L{2} -L{3} {4}{5} {6}{7} -B{8} "$@"\n'.format(
+                        os.path.join(self.spec['llvm'].prefix.bin, 'flang'),
```

Comment:
You can actually use `self.spec.prefix.bin.flang` here, no need for `os.path.join`. Same elsewhere in the `package.py`

Reply:
Fixed